libs/designsystem/src/lib/components/text-link/text-link.component.spec.ts

```diff
@@ -0,0 +1,175 @@
+      expect(anchor).toHaveComputedStyle({ color: getColor('black') });
+    });
+
+    it('styled with an underline', () => {
+      const anchor = spectator.queryHost<HTMLAnchorElement>('a');
+      expect(anchor).toHaveComputedStyle({ 'text-decoration-line': 'underline' });
+    });
+
+    it('styled with a pointer as the cursor', () => {
+      const textlink = spectator.queryHost<HTMLElement>('kirby-text-link');
+      expect(textlink).toHaveComputedStyle({ cursor: 'pointer' });
+    });
+  });
+  describe('with external links', () => {
+    beforeEach(() => {
+      spectator = createHost(
+        `<kirby-text-link  text='Some Link' link='https://angular.io/api/router/RouterLink'></kirby-text-link>`
+      );
+    });
+    it('should have an icon', () => {
+      const icon = spectator.queryHost<HTMLElement>('kirby-icon');
+      expect(icon).toBeTruthy();
+    });
+    it('should have icon named [link]', () => {
+      const icon = spectator.queryHost<HTMLElement>('kirby-icon');
+
+      const iconName = icon.getAttribute('name');
+      expect(iconName).toBe('link');
+    });
+
+    it('should render the icon to the right from the text', () => {
+      const icon = spectator.queryHost<HTMLElement>('kirby-icon');
+      const text = spectator.queryHost<HTMLElement>('a');
+      const textPosition = text.getBoundingClientRect();
+      const iconPosition = icon.getBoundingClientRect();
+      expect(textPosition.right).toBeLessThan(iconPosition.right);
+    });
+  });
+  describe('size and padding', () => {
+    interface TestCase {
+      size?: string;
+      expected: {
+        fontSize: string;
+        fontLineHeight: string;
+        iconPaddingLeft: string;
+        iconPaddingRight: string;
+        iconFontSize: string;
+        iconVerticalAlign: string;
+      };
+    }
+
+    const testCases: TestCase[] = [
+      {
+        size: 'md',
+        expected: {
+          fontSize: 'n',
+          fontLineHeight: 'n',
+          iconPaddingLeft: 'xxxs',
+          iconPaddingRight: 's',
+          iconFontSize: 'm',
+          iconVerticalAlign: 'middle',
+        },
+      },
+      {
+        size: 'sm',
+        expected: {
+          fontSize: 's',
+          fontLineHeight: 's',
+          iconPaddingLeft: 'xxxs',
+          iconPaddingRight: 's',
+          iconFontSize: 'm',
+          iconVerticalAlign: 'middle',
+        },
+      },
+      {
+        size: 'xs',
+        expected: {
+          fontSize: 'xs',
+          fontLineHeight: 'xs',
+          iconPaddingLeft: 'xxxxs',
+          iconPaddingRight: 's',
+          iconFontSize: 's',
+          iconVerticalAlign: 'middle',
+        },
+      },
+      {
+        size: '',
+        expected: {
+          fontSize: 'n',
+          fontLineHeight: 'n',
+          iconPaddingLeft: 'xxxs',
+          iconPaddingRight: 's',
+          iconFontSize: 'm',
+          iconVerticalAlign: 'middle',
+        },
+      },
+    ];
+    testCases.forEach((scenario) => {
```

Comment:
Consider readability:
```suggestion
    ];
    
    testCases.forEach((scenario) => {
```